dist/api_data.py

```diff
@@ -1430,6 +1430,9 @@ def __ge__(self, other):
             type='boolean'),
         Config('file_metadata', '', r'''
             the file configuration extracted from the metadata of the export database'''),
+        Config('metadata_file', '', r'''
```

Comment:
I would say "File that contains all the relevant metadata information for the URI to import"

Reply:
Changed.